code/game/objects/structures/window.dm

```diff
@@ -107,7 +107,8 @@
 		if(user && istype(user))
 			user.count_niche_stat(STATISTICS_NICHE_DESTRUCTION_WINDOWS, 1)
 			SEND_SIGNAL(user, COMSIG_MOB_DESTROY_WINDOW, src)
-			user.visible_message(SPAN_DANGER("[user] smashes through [src][AM ? " with [AM]":""]!"))
+			for(var/mob/living/carbon/viewers_in_range in orange(7, src))
+				to_chat(viewers_in_range, SPAN_WARNING("[user] smashes through the [src][AM ? " with [AM]":""]!"))
```

Comment:
When you loop over something the variable you use to loop (in this case `viewers_in_range`) should be generally singular